src/lib/molecules/IntroKast.svelte

```diff
@@ -1,89 +1,81 @@
 <div class="top-info">
-    <div class="wrapper_info">
-        <div class="container_info">
```

Comment:
je gebruikt hier veel div is het mogelijk om het iets semantischer te schrijven mabey kan de container div een article zijn  enz.